pkg/kubelet/dockershim/network/hostport/fake_iptables.go

```diff
@@ -148,14 +148,14 @@ func (f *fakeIPTables) ensureRule(position utiliptables.RulePosition, tableName
 		return true, nil
 	}
 
-	if position == utiliptables.Prepend {
```

Comment:
Ideally this would be in a sub-pkg, not linked into prod code.  Not going to hold this up for that.